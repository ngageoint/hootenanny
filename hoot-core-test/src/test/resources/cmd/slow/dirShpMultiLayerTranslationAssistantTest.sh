#!/bin/bash
set -e

export TA_FILE=calizip
export TA_IN=test-files/translation_assistant
export TA_OUT=test-output/cmd/translation_assistant_ogr2osm
mkdir -p $TA_OUT

hoot --ogr2osm $TA_IN/$TA_FILE-translation.js $TA_OUT/$TA_FILE.osm $TA_IN/$TA_FILE
hoot --is-match --ignore-uuid $TA_OUT/$TA_FILE.osm $TA_IN/$TA_FILE.osm

