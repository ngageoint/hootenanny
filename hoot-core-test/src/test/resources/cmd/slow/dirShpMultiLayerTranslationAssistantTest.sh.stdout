13:18:02.522 INFO  ...p/hoot/core/cmd/Ogr2OsmCmd.cpp( 128) layers: [2]{cali-fake-points, cali-test}
13:18:02.526 INFO  ...p/hoot/core/cmd/Ogr2OsmCmd.cpp( 177) Reading: test-files/translation_assistant/calizip cali-fake-points
Loading test-files/translation_assistant/calizip cali-fake-points 0 / 4   
13:18:02.527 INFO  ...p/hoot/core/cmd/Ogr2OsmCmd.cpp( 177) Reading: test-files/translation_assistant/calizip cali-test
Loading test-files/translation_assistant/calizip cali-test 0 / 357   
Reprojecting 0 / 2343         Reprojecting 1000 / 2343         Reprojecting 2000 / 2343         
13:18:02.616 INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::MergeNearbyNodes
13:18:02.616 INFO  .../core/ops/MergeNearbyNodes.cpp(  73) MergeNearbyNodes start
MergeNearbyNodes 0 0    MergeNearbyNodes 1000 732    MergeNearbyNodes 1402 elapsed: 13ms        
13:18:02.629 INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::BuildingPartMergeOp
Ways: 0 / 357        Ways: 357 / 357        
Relations: 0 / 0        
Combining Parts: 0 / 0        
Reprojecting 0 / 1402         Reprojecting 1000 / 1402         
